tests/test_integration/stencil_definitions.py

```diff
@@ -307,7 +325,7 @@ def local_var_inside_nested_conditional(in_storage: Field3D, out_storage: Field3
     with computation(FORWARD), interval(2, None):
         if in_storage[0, 0, 0] < 0:
             local_var = 6
-            out_storage[0, 0, 0] = local_var + mid_storage
+            out_storage[0, 0, 0] = local_var
```

Comment:
This test actually had an error to begin with, but was never caught before. This fixes it.